Kickstarter-iOS/Views/PledgeCTAContainerView.swift

```diff
@@ -95,8 +105,15 @@ final class PledgeCTAContainerView: UIView {
   override func bindViewModel() {
     super.bindViewModel()
 
-    self.amountAndRewardTitleStackView.rac.hidden = self.vm.outputs.stackViewIsHidden
-    self.amountOrRewardLabel.rac.text = self.vm.outputs.rewardTitle
+    self.vm.outputs.buttonTitleTextColor
+      .observeForUI()
+      .observeValues { [weak self] textColor in
+        self?.pledgeCTAButton.setTitleColor(textColor, for: .normal)
```

Comment:
Just to note, by setting this we don't have any highlight state colours. Something I mentioned to @dannyalright is that we need highlight state colours for the buttons. This can probably just be like opacity `0.5` -> `0.8` for the textColor and a slightly darker colour for the button background which we can potentially just do programmatically.

Reply:
chatted on slack w/ @justinswart about doing a future PR addressing this